client/apps/admin-portal/src/app/app.component.html

```diff
@@ -1 +1,18 @@
-<router-outlet></router-outlet>
\ No newline at end of file
+<sb-toolbar [sidenav]="sidenav"></sb-toolbar>
+
+<mat-sidenav-container class="vh-100">
```

Comment:
I see two `mat-sidenav`'s, not sure which one is being used.
There is one here and in the `SidenavComponent`.